packages/protocol/truffle.js

```diff
@@ -28,7 +28,7 @@ const networks = {
     network_id: '1101',
     from: DEVELOPMENT_FROM,
     gasPrice: 0,
-    gas: 6500000,
+    gas: 6700000,
```

Comment:
Why did this change? Does it deserve a comment?

Reply:
Increased gas limit so that Governance could be deployed. Noted in the PR description; should I also comment in the code?